docs/src/main/asciidoc/security-identity-providers.adoc

```diff
@@ -9,18 +8,18 @@ include::_attributes.adoc[]
 :diataxis-type: concept
 :categories: security
 
-In the Quarkus Security framework, identity providers play a key role in authentication and authorization, providing services for storing and verifying user identities.
-The Jakarta Persistence `IdentityProvider` creates a `SecurityIdentity` instance used during user authentication to verify and authorize access requests, making your Quarkus application secure.
+In the Quarkus Security framework, identity providers play a crucial role in authentication and authorization, providing services for storing and verifying user identities.
```

Comment:
```suggestion
In the Quarkus Security framework, identity providers play a crucial role in authentication and authorization by  verifying user identities.
```

Reply:
`providing services for storing and verifying user identities` does not sound quite right, one of the identity providers uses the internally stored identities to support their verification but at the Quarkus security architecture level, it is an implementation detail of the (JPA) identity provider

Reply:
@sberyozkin - Fixed. Thanks.